src/components/LeftDrawer/LeftDrawer.module.css

```diff
@@ -60,9 +60,9 @@
 .leftDrawer .profileContainer {
   border: none;
   width: 100%;
+  padding: 2.1rem 0.5rem;
   height: 52px;
   border-radius: 8px;
-  background-color: var(--bs-white);
```

Comment:
Why is this changed?

Reply:
The above changes were made to match styling of profileContainer from `LeftDrawerOrg`  (similar component for org screen)
https://github.com/GlenDsza/talawa-admin/blob/030647b0a2511daa43b185e5a8d4c9aedf990280/src/components/LeftDrawerOrg/LeftDrawerOrg.module.css#L86C1-L95C2

The resulting grey background highlights the profileContainer from drawer white background
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/109027247/48a70b66-3330-483d-9a35-594576625fa1)

Reply:
Ah. Ohk.